packages/renovate-config/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@researchgate/renovate-config",
+  "version": "1.3.0",
+  "renovate-config": {
+    "default": {
+      "extends": [
+        "config:base",
+        "default:preserveSemverRanges",
+        "default:rebaseStalePrs",
+        "schedule:nonOfficeHours"
```

Comment:
Non office hours still happens every day. Are we sure we want it?

Reply:
It usually works quite well for small- to medium-sized projects (especially then used with Spire). I'd suggest to try this schedule and adjust later if needed. It's still possible to override per-project.